website/docs/tutorials/08-cairo-1-support/03-cheatcodes/declare.md

```diff
@@ -0,0 +1,10 @@
+# `declare`
+
+```cairo
+fn declare(contract: felt) -> Result::<felt, felt> nopanic;
+```
+
+Declares a contract given a contract name defined the [protostar.toml](../../04-configuration-file.md) configuration
```

Comment:
We should tell the user that this cheatcode only supports cairo 1 now, and reference to declaration of old ones.